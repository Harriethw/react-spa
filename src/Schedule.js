import React, { Component } from "react";
import Card from 'react-bootstrap/Card';
 
class Schedule extends Component {
  render() {
    return (
      <Card.Body>
        <Card.Title>Welcome!</Card.Title>
        <Card.Text>Cras facilisis urna ornare ex volutpat, et
        convallis erat elementum. Ut aliquam, ipsum vitae
        gravida suscipit, metus dui bibendum est, eget rhoncus nibh
        metus nec massa. Maecenas hendrerit laoreet augue
        nec molestie. Cum sociis natoque penatibus et magnis
        dis parturient montes, nascetur ridiculus mus.</Card.Text>
 
        <Card.Text>Duis a turpis sed lacus dapibus elementum sed eu lectus.</Card.Text>

        <Card.Subtitle>09:00-10:00</Card.Subtitle>
        <Card.Text>first talk</Card.Text>

        <Card.Subtitle>09:00-10:00</Card.Subtitle>
        <Card.Text>second talk</Card.Text>

        <Card.Subtitle>09:00-10:00</Card.Subtitle>
        <Card.Text>third talk</Card.Text>

        <Card.Subtitle>09:00-10:00</Card.Subtitle>
        <Card.Text>fourth talk</Card.Text>
      </Card.Body>
    );
  }
}
 
export default Schedule;