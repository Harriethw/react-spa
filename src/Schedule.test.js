import React from 'react';
import { shallow } from 'enzyme';
import Schedule from './Schedule';

describe('Schedule', () => {
  it('should render correctly ', () => {
    const component = shallow(<Schedule />);
  
    expect(component).toMatchSnapshot();
  });
});