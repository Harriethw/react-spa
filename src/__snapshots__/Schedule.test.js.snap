// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schedule should render correctly  1`] = `
<CardBody>
  <CardTitle>
    Welcome!
  </CardTitle>
  <CardText>
    Cras facilisis urna ornare ex volutpat, et convallis erat elementum. Ut aliquam, ipsum vitae gravida suscipit, metus dui bibendum est, eget rhoncus nibh metus nec massa.
  </CardText>
  <CardTitle>
    Schedule
  </CardTitle>
  <CardSubtitle>
    09:00-10:00
  </CardSubtitle>
  <CardText>
    first talk
  </CardText>
  <CardSubtitle>
    09:00-10:00
  </CardSubtitle>
  <CardText>
    second talk
  </CardText>
  <CardSubtitle>
    09:00-10:00
  </CardSubtitle>
  <CardText>
    third talk
  </CardText>
  <CardSubtitle>
    09:00-10:00
  </CardSubtitle>
  <CardText>
    fourth talk
  </CardText>
</CardBody>
`;
